Makefile

```diff
@@ -7,11 +7,11 @@ DESTDIR=/usr/local
 # Used to populate version variable in main package.
 VERSION=$(shell git describe --match 'v[0-9]*' --dirty='.m' --always)
```

Comment:
`m[modified]`

I have no clue where I picked up this convention but I've used it in several places. The biggest benefit is obfuscation.